srcd/dashboards/gitbase/welcome.json

```diff
@@ -3,7 +3,7 @@
tp://127.0.0.1:8088/superset/sqllab)\\n- [List and create charts](http://127.0.0.1:8088/chart/list/)\\n- [List and create dashboards](http://127.0.0.1:8088/dashboard/list/)\\n- [Explore Universal ASTs](http://127.0.0.1:8088/bblfsh/explore/)\\n\\n_Note: You may also use the \\\"New\\\" button on the top right of the interface._\\n\\n### Check examples for inspiration on new use cases and metrics\\n\\n- [SQL query examples](https://docs.sourced.tech/community-edition/usage/examples)\\n- [Universal AST query examples](https://docs.sourced.tech/babelfish/using-babelfish/uast-querying)\\n\\n### Read the documentation for detailed information\\n\\n- [Introduction: overview & index](https://docs.sourced.tech/community-edition/)\\n- [Quick-start: dependencies, installation, initialization & exploration of datasets](https://docs.sourced.tech/community-edition/quickstart)\\n- [Usage: query examples, working with multiple datasets, CLI commands](https://docs.sourced.tech/community-edition/usage)\\n- [Learn more: architecture, Universal ASTs](https://docs.sourced.tech/community-edition/advanced)\\n\\n### Join the community, share use cases, ask questions, report issues\\n\\n- [source{d} CE project](https://github.com/src-d/sourced-ce/)\\n- [Community Forum](https://forum.sourced.tech)\\n- [Slack Community](http://bit.ly/src-d-community)\\n- [GitHub](https://github.com/src-d/)\\n\\n## Further references\\n\\nIn the following places you can find further information:\\n\\n- [source{d} website](https://www.sourced.tech/)\\n  - [source{d} Community Edition](https://sourced.tech/features/)\\n  - [source{d} Enterprise Edition](https://sourced.tech/enterprise/)\\n  - [Company](https://sourced.tech/company)\\n  - [Blog](https://blog.sourced.tech/)\\n- [Documentation](https://docs.sourced.tech/)\\n  - [Babelfish/UASTs](https://docs.sourced.tech/babelfish/)\\n  - [Apache Superset (external)](https://superset.incubator.apache.org/)\\n- [Open Source](https://github.com/src-d/)\\n  - [source{d} CE project](https://github.com/src-d/sourced-ce/)\\n  - [Machine Learning on Code](https://github.com/src-d/awesome-machine-learning-on-source-code)\\n  - [Datasets](https://github.com/src-d/datasets)\\n  - [gitbase](https://github.com/src-d/gitbase)\\n- [Community](https://sourced.tech/community)\\n  - [Newsletter](https://go.sourced.tech/newsletter)\\n  - [Slack Community](http://bit.ly/src-d-community)\\n  - [Forum](https://forum.sourced.tech)\\n  - [Twitter](https://twitter.com/sourcedtech)\\n  - [YouTube](https://www.youtube.com/sourced/)\\n  - [Email](mailto:hello@sourced.tech)\",\"height\":196,\"width\":12},\"parents\":[\"ROOT_ID\",\"GRID_ID\",\"ROW-he3mkLfLAE\"],\"type\":\"MARKDOWN\"}, \"ROOT_ID\": {\"children\": [\"GRID_ID\"], \"id\": \"ROOT_ID\", \"type\": \"ROOT\"}, \"ROW-he3mkLfLAE\": {\"children\": [\"MARKDOWN-ENl8jvIbic\"], \"id\": \"ROW-he3mkLfLAE\", \"meta\": {\"background\": \"BACKGROUND_TRANSPARENT\"}, \"parents\": [\"ROOT_ID\", \"GRID_ID\"], \"type\": \"ROW\"}}",
```

Comment:
Agreed with @carlosms on the height of the markdown widget, double scrollbars are not intuitive in either browsers/OSes that hide it by default or ugly/confusing in those which do by default.

It is also helpful to add some extra room/space in the height of the widget so that in case there's some unexpected line-breaks it doesn't break overflowing/scrolling.